docs/dimensions.md

```diff
@@ -182,6 +182,21 @@ This should only be called from native code by sending the didUpdateDimensions e
 
 **Parameters:**
 
-| Name | Type   | Required | Description                              |
-| ---- | ------ | -------- | ---------------------------------------- |
-| dims | object | Yes      | string-keyed object of dimensions to set |
+| Name                                                      | Type   | Description                              |
+| --------------------------------------------------------- | ------ | ---------------------------------------- |
+| dims <div className="label basic required">Required</div> | object | string-keyed object of dimensions to set |
+
+---
+
+### DimensionsValue
+
+| Type   |
+| ------ |
+| object |
+
+**Properties:**
+
+| Name   | Type                                            | Description                                    |
+| ------ | ----------------------------------------------- | ---------------------------------------------- |
+| window | [addEventListener](dimensions#addeventlistener) | Defines Size of the visible Application window |
+| screen | [addEventListener](dimensions#addeventlistener) | Size of the device's screen                    |
```

Comment:
The correct type of those properties is [`DisplayMetrics`](https://github.com/facebook/react-native/blob/main/Libraries/Utilities/NativeDeviceInfo.js#L22:L27) which also need to be added in the "Type Definitions" section.

By replacing the type in listener I have in mind the portion of description, which now can be replaced with link to the `DimensionsValue` type.

<img width="627" alt="Screenshot 2021-07-22 162916" src="https://user-images.githubusercontent.com/719641/126656617-8b95aa92-3ea8-4f03-bda6-71517e8335b5.png">

Reply:
hi @Simek  i took this example table https://reactnative.dev/docs/next/appregistry#registry, for add the values `DimensionsValue`, is that correct ?

Reply:
Unfortunately not, but it's not a problem. Let check out this branch and help you a bit. 🙂

Reply:
i tried to put 4 types for every name properties like `window` but i'm no sure looks weird.

Reply:
What you have done it's fine, there was a bit confusion on the review request. Also those types are not that straight forward. 

To be honest, probably the small Flow typing refactor in React Native won't hunt. I will try too look at this during weekend. In the other hand people should use `useWindowDimensions` hook instead and this API in a long term is deems to be deprecated, so maybe it's not that important.

Currently we are missing the Android specific optional keys for the dimensions payload, but to document this properly, the small refactor in core will be needed. 

The PR should be now ready to merge, thank you for the updates and work on the docs conversion to the new format! 👍

Reply:
you're right i think most of the people use `useWindowDimensions`. thanks for the review i try to add another required labels for the API docs. 😃